how-to-guides/how-it-works/notebooks/Neptune_API_Tour.ipynb

```diff
@@ -256,7 +254,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "! pip install scikit-plot==0.3.7 tensorflow==2.4.1"
+    "! pip install scikit-plot tensorflow"
```

Comment:
Tensorflow and PTL are about to undergo some changes.

Let's monitor them closely, especially because we are unpinning their dependencies.

Reply:
yup, unpinning will let us know if something breaks in their latest releases